api/swagger-spec/v1.json

```diff
@@ -5971,7 +5971,7 @@
         "allowMultiple": false
        },
        {
-        "type": "",
+        "type": "string",
```

Comment:
Please also note that it assumes the values are comma-separated so the server should expect comma-separated values. 

Swagger Spec 2.0 has more options on the format. Please refer to https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#fixed-fields-7 => `collectionFormat` for more information